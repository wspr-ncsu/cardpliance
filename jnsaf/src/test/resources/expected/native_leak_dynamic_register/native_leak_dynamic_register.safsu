`Lorg/arguslab/native_leak_dynamic_register/MainActivity;.send:(Ljava/lang/String;)V`:
;