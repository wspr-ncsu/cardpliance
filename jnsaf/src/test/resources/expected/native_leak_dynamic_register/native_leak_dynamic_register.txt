Lorg/arguslab/native_leak_dynamic_register/MainActivity;.send:(Ljava/lang/String;)V -> _SINK_ 1